src/update-manager.js

```diff
@@ -621,6 +627,7 @@ class UpdateManager {
   update (deletions, insertions, callback, secondTry) {
     try {
       var kb = this.store
+      var i
```

Comment:
Why declaring `i` there? It seems like it isn't used outside of the `for` loops further down, and that there is no reason why it shouldn't be scoped to those loops?

Reply:
Fixed all declarations of i and j to be local 'let'.